### PR TITLE
Code examples for episode 006

### PR DESCRIPTION
**Pull request by int3l**
I axed some of the unrelated code potion in the `t.py` examples in order to make it more concise and simple.
Let me know if you want the whole codebase from the previous episode.

### CODE REVIEW

File: ep006/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+### Python
+
+```python
+# python3.8
+Location()
+class C:
+    @property
+    def __class__(self):
+        print('return custom type')
```
Comment (int3l) [inline ep006/README.md:15]: Added debug print for better visibility of code execution. Let me if you want this to be removed as well.

File: ep006/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        return int
+
+C().__class__
+C().__class__.__name__
```
Comment (int3l) [inline ep006/README.md:19]: I think that you meant to show the `.__class__.__name__` attribute, so I added it as well.

File: ep006/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# [python type(x).__name__ vs x.__class__.__name__ (intermediate)](https://youtu.be/6rAIttnm3Fs)
+
+Very short one, I explain why I prefer type(x).__name__ vs x.__class__.__name__.  thanks to @alikus for the question!
```
Comment (asottile) [inline ep006/README.md:3]: these will need backticks around them or they'll end up italicized

**Review by asottile — APPROVED** [review]

File: ep006/README.md (hunk 3 above)
Comment (int3l) [reply ep006/README.md]: Yep, I will extract the descriptions manually from now on. They need additional processing.

**Pull request merged**